app/controllers/application_controller.rb

```diff
@@ -1,4 +1,16 @@
-# ApplicationController
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+  before_action :set_locale
+
+  def change_locale
+    current_user.update_attribute(:locale, params[:locale])
+    redirect_back(fallback_location: user_session)
+  end
+
+  private
+
+  def set_locale
+    I18n.locale = (current_user.locale unless current_user.nil?) ||
```

Comment:
should be single line

Reply:
Method set_locale now have a single line